### ISSUE TITLE
On new install, core folders not created in "games"

### ISSUE THREAD

**Issue by dshadoff** [MiSTer-devel/Updater_script_MiSTer#42]
On a new install, core folders are created in the root folder, not subordinate to "games"

**Comment by Locutus73** [comment]
Yes, it's the default behaviour now: MiSTer supports both root and games with no philosophical preference (actually /media/fat/<core> is choosen over /media/fat/games/<core> when both exist).
The current behaviour of the updater is using /media/fat/games only when it exists and there's at least one file inside it (or in any subdir).
Since most users seem to prefer games I'm going to change the updater preference, using games unless there's some non empty core dir in the root.
Problem is that, while checking if games is not empty is trivial, checking for core dirs in the root is not so trivial (which cores?).

**Comment by dshadoff** [comment]
We can consider that the minority of people who might prefer not to put their games into /media/fat/games , will have an empty /media/fat/games directory.  Or perhaps they have already removed it.
So, existence of /media/fat/games is a key discriminating factor.

I had encountered this on a "new install/rebuild" instance - where one could keep or delete the /media/fat/games folder to direct the behavior - but the existing behavior could be kept for all other situations.

In other words, if games exists but is empty, count the directories in /media/fat to determine whether this is a blank, fresh install.

**Comment by Locutus73** [comment]
Implemented in v4.0.6
https://github.com/MiSTer-devel/Updater_script_MiSTer/commit/53d4ed50a582ba7833d68de267faa58b63c51e77
